thinc_apple_ops/blas.pyx

```diff
@@ -37,9 +37,14 @@ cdef extern from "Accelerate/Accelerate.h":
 cpdef np.ndarray gemm(float[:, ::1] A, float[:, ::1] B, bint trans1=False, bint trans2=False): 
     cdef int nM = A.shape[0] if not trans1 else A.shape[1]
     cdef int nK = A.shape[1] if not trans1 else A.shape[0]
+    cdef int nK_b = B.shape[0] if not trans2 else B.shape[1]
     cdef int nN = B.shape[1] if not trans2 else B.shape[0]
     cdef np.ndarray out = numpy.empty((nM, nN), dtype="f")
 
+    if nK != nK_b:
+        msg = "Shape mismatch for blis.gemm: (%d, %d), (%d, %d)"
```

Comment:
Not `blis`, I don't know how much detail you want to get into in this message:

```suggestion
        msg = "Shape mismatch for gemm: (%d, %d), (%d, %d)"
```

Reply:
Ah right! The concrete implementation does not matter, so I changed it to just `gemm` as you suggested.